project/ProjectPlugin.scala

```diff
@@ -1,48 +1,114 @@
-      addSbtPlugin("de.heikoseeberger"  % "sbt-header"             % "1.8.0"),
+      addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.9.3" commonExcludes),
+      addSbtPlugin("com.eed3si9n"       % "sbt-unidoc"             % "0.4.1"),
+      addSbtPlugin("com.github.gseitz"  % "sbt-release"            % "1.0.6"),
+      addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"           % "2.0"),
+      addSbtPlugin("com.jsuereth"       % "sbt-pgp"                % "1.1.0-M1"),
+      addSbtPlugin("com.typesafe.sbt"   % "sbt-ghpages"            % "0.6.2"),
+      addSbtPlugin("com.typesafe.sbt"   % "sbt-site"               % "1.3.0"),
+      addSbtPlugin("pl.project13.scala" % "sbt-jmh"                % "0.2.27"),
+      addSbtPlugin("org.scalastyle"     %% "scalastyle-sbt-plugin" % "1.0.0"),
+      addSbtPlugin("org.scoverage"      % "sbt-scoverage"          % "1.5.1"),
+      addSbtPlugin("org.scala-js"       % "sbt-scalajs"            % "0.6.19"),
+      addSbtPlugin("de.heikoseeberger"  % "sbt-header"             % "3.0.1"),
       addSbtPlugin("com.eed3si9n"       % "sbt-buildinfo"          % "0.7.0"),
-      addSbtPlugin("com.geirsson"       % "sbt-scalafmt"           % "0.6.8"),
-      addSbtPlugin("io.get-coursier"    % "sbt-coursier"           % "1.0.0-RC9"),
-      addSbtPlugin("com.47deg"          % "sbt-dependencies"       % "0.1.1"),
-      addSbtPlugin("com.47deg"          % "sbt-microsites"         % "0.6.1" exclude("org.tpolecat", "tut-plugin"))
-    ) ++
-      ScriptedPlugin.scriptedSettings ++ Seq(
-      scriptedDependencies := (compile in Test) map { _ =>
-        Unit
-      },
+      // addSbtPlugin("com.lucidchart"     % "sbt-scalafmt"  % "1.10"),
+      // addSbtPlugin("com.geirsson"       % "sbt-scalafmt"  % "1.2.0"),
+      libraryDependencies ++= {
+        val sbtVersionValue       = (sbtVersion in pluginCrossBuild).value
+        val sbtBinaryVersionValue = (sbtBinaryVersion in pluginCrossBuild).value
+
+        val scalaBinaryVersionValue = (scalaBinaryVersion in update).value
+
+        sbtVersionValue match {
+          case sbtV.`0.13` =>
+            Seq(
+              Defaults.sbtPluginExtra(
+                "org.tpolecat" % "tut-plugin" % "0.5.3",
+                sbtBinaryVersionValue,
+                scalaBinaryVersionValue),
+              Defaults.sbtPluginExtra(
+                "com.geirsson" % "sbt-scalafmt" % "0.6.8",
+                sbtBinaryVersionValue,
+                scalaBinaryVersionValue)
+            )
+          case sbtV.`1.0` =>
+            Seq(
+              Defaults.sbtPluginExtra(
+                "org.tpolecat" % "tut-plugin" % "0.6.0",
+                sbtBinaryVersionValue,
+                scalaBinaryVersionValue),
+              Defaults.sbtPluginExtra(
+                "com.geirsson" % "sbt-scalafmt" % "1.2.0",
+                sbtBinaryVersionValue,
+                scalaBinaryVersionValue)
+            )
+        }
```

Comment:
https://github.com/tpolecat/tut
0.6.x -> sbt 1.x
0.5.x -> sbt 0.13.x

`sbt-scalafmt` -> I want to eventually switch to `neo-sbt-scalafmt` but I didn't then understand why scala 2.12 dependencies were getting pulled in for `scalafmt-core`. I currently kept different versions fot `sbt-scalafmt` to simply keep status quo.

Reply:
Sounds good, let's open an issue to address that migration later on.

Reply:
Switch to `neo-sbt-scalafmt` : https://github.com/47deg/sbt-org-policies/issues/511